### DISCUSSION TITLE
24.03.15(목) 회의

### DISCUSSION THREAD

**Discussion by Shinjiwoooo** [NC4-FinalProject/NC4_Final_Project_2.14#9] [Announcements]
# 회의 내용

##  개발 일정 및 진행 순서
-  내일 Rlease 브랜치 생성 후 배치 까지 할 예정 (태현)
-  3월 21일 프론트(앞단) 개발 완료 
-  4월  5일  프론트,백엔드 전체 개발 완료 
-  4월  9일  Rlease서버운영 및 도메인 구입 예정

##  Git Commit Message Convention 
- notion에 git commit Message Convention수정된 내용 자세 읽어보기
https://screeching-wilderness-818.notion.site/Git-Commit-Message-Convention-779248b1ee8e4ccd901bb1d30d6d558f

- 예시
![image](https://github.com/NC4-FinalProject/NC4_Final_Project_2.14/assets/93236136/77306a3b-bdeb-4014-8f79-48e82e9f059b)

## Notion

+ 컴포넌트 명세서 와 API명세서 작성 https://drive.google.com/drive/folders/1wpulH2l-fqbVc54gC-x8Kw3g4DQuewfT
(컴포넌트 생성 및 api 생성시 바로 작성)

## Git 올릴 때 주의사항
- conflict(충돌) 났을때 무조건 내파일과 충돌난 상대를 불러서 같이 수정 
- main 에 최종적으로 병합 후 본인 브랜치를 기점으로   merge 받기 

## 개발노트 
- 본인이 개발하면서 힘들었던 점이나, 오류 해결 과정, 피드백 등 velog,tistory,notion 등을 이용하여 작성하기
(나중에 발표시 회고록 또는 피드백 시  사용가능)

- 예시 https://velog.io/@lsg6542/%EC%BF%BC%EB%A6%AC-%EC%98%A4%EB%A5%98-%ED%95%B4%EA%B2%B0-%EA%B3%BC%EC%A0%95

## 다음 회의 주제
-  각자 맡은 기능 앞단 컴포넌트 개발  및 merge 작업
-  도메인 이름 정할 것 생각해보기

**Comment by Selah-on-Github** [comment]
내용 확인했습니다.

**Comment by Robbie-Kim** [comment]
확인완료입니다
